sentry/src/main/java/io/sentry/SentryEnvelopeItemHeader.java

```diff
@@ -42,23 +43,36 @@ public int getLength() {
       final @NotNull SentryItemType type,
       int length,
       final @Nullable String contentType,
-      final @Nullable String fileName) {
+      final @Nullable String fileName,
+      final @Nullable String attachmentType) {
```

Comment:
`m`: I see that we often pass `null` for `attachmentType`. Does it maybe make sense to define an overload so we don't need to pass `null`?

Reply:
makes sense